doc/tutorials/objdetect/traincascade.markdown

```diff
@@ -4,66 +4,28 @@ Cascade Classifier Training {#tutorial_traincascade}
-    false alarms and other information.
-
-Since opencv_haartraining is an obsolete application, only opencv_traincascade will be described
-futher. opencv_createsamples utility is needed to prepare a training data for opencv_traincascade,
-so it will be described too.
-
-Training data preparation
--------------------------
-
-For training we need a set of samples. There are two types of samples: negative and positive.
-Negative samples correspond to non-object images. Positive samples correspond to images with
-detected objects. Set of negative samples must be prepared manually, whereas set of positive samples
-is created using opencv_createsamples utility.
+ * If you come accross any tutorial mentioning the old opencv\_haartraining tool <i>(which is depricated and still using the OpenCV1.x interface)</i>, then please ignore that tutorial and stick to the [opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/traincascade) tool. This tool is a is a newer version, written in C++ in accordance to the OpenCV 2.x and OpenCV 3.x API. The [opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/traincascade) supports both HAAR like wavelet features @cite Viola01 and LBP (Local Binary Patterns) @cite Liao2007 features. LBP features yield integer precision in contrast to HAAR features, yielding floating point precision, so both training and detection with LBP are several times faster then with HAAR features. Regarding the LBP and HAAR detection quality, it mainly depends on the training data used and the training parameters selected.It's possible to train a LBP-based classifier that will provide almost the same quality as HAAR-based one, within a percentage of the training time.
+
+ * The newer cascade classifier detection interface from OpenCV 2.x and OpenCV 3.x (see CascadeClassifier class in objdetect module) supports working with both old and new model formats. [opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/traincascade) can even save (export) a trained cascade in the older format if for some reason you are stuck using the old interface. At least training the model could then be done in the most stable interface.
+
+ * The [opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/traincascade) application can use TBB for multi-threading. To use it in multicore mode OpenCV must be built with TBB support enabled.
+
+Preparation of the training data
+--------------------------------
+
+For training a boosted cascade of weak classifiers we need a set of positive samples (containing actual objects you want to detect) and a set of negative images (containing everything you do not want to detect). The set of negative samples must be prepared manually, whereas set of positive samples is created using the [opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/createsamples) application.
```

Comment:
Should be `create_samples`: `[opencv\_traincascade](https://github.com/opencv/opencv/tree/master/apps/createsamples)`